### PR TITLE
[OperationalInsights] Bug fix: when updating a cluster, it's SKU is set even if no value was passed

### PR DESCRIPTION
**Pull request by dabenhamMic**
…ven if no value was passed

added null check for KeyVaultProperties to prevent null ref exception

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by wyunchi-ms** [comment]
Please fix the static analysis exceptions.

**Comment by dabenhamMic** [comment]
Hi @wyunchi-ms, can we move forward with this PR?

File: src/OperationalInsights/OperationalInsights/ChangeLog.md [hunk 1]
```diff
@@ -16,8 +16,10 @@
     ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
     * Overview of change #1
         - Additional information about change #1
--->
```
Comment (BethanyZhou) [inline src/OperationalInsights/OperationalInsights/ChangeLog.md:19]: --> is needed. Please add it back

File: tools/StaticAnalysis/Exceptions/Az.OperationalInsights/BreakingChangeIssues.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
+"Az.OperationalInsights","UpdateAzureOperationalInsightsClusterCommand.cs","Update-AzOperationalInsightsCluster","0","2020","The parameter SkuCapacity was changed from long to nullable long to prevent a scenario which no value is passed and 0 value is being used","The cmdlet 'Update-AzOperationalInsightsCluster' no longer supports the type 'System.Int64' for parameter 'SkuCapacity'."",""Change the type for parameter 'SkuCapacity' back to 'System.Int64'."
```
Comment (BethanyZhou) [inline tools/StaticAnalysis/Exceptions/Az.OperationalInsights/BreakingChangeIssues.csv:2]:
```suggestion
"Az.OperationalInsights","Microsoft.Azure.Commands.OperationalInsights.Clusters.UpdateAzureOperationalInsightsClusterCommand","Update-AzOperationalInsightsCluster","0","2020","The cmdlet 'Update-AzOperationalInsightsCluster' no longer supports the type 'System.Int64' for parameter 'SkuCapacity'.","Change the type for parameter 'SkuCapacity' back to 'System.Int64'."
```

**Review by BethanyZhou — APPROVED** [review]
LGTM

**Pull request merged**